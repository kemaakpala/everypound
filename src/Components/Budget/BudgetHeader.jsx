import Grid from '@material-ui/core/Grid';
import Avatar from '@material-ui/core/Avatar';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
const useStyles = makeStyles((theme) => ({
  groupLabel: {
    display: 'flex',
    flex: 1,
    alignItems: 'center',
    '& p': {
      color: 'red',
      paddingLeft: theme.spacing(1)
    }
  }
}));
const BudgetHeader = ({ groupTitle }) => {
  const classes = useStyles();
  return (
    <Grid item xs={12} style={{ border: '1px solid red' }}>
      <Grid container spacing={1} alignItems="center" style={{ border: '1px solid blue' }}>
        <Grid item xs={4}>
          <div className={classes.groupLabel}>
            <Avatar>{groupTitle[0].toUpperCase()}</Avatar>
            <Typography>{groupTitle}</Typography>
          </div>
        </Grid>
        <Grid item xs={4}><Typography>Planned</Typography></Grid>
        <Grid item xs={4}><Typography>Actual</Typography></Grid>
      </Grid>
    </Grid>
  )
}

export default BudgetHeader;