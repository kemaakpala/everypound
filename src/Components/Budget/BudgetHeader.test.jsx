import BudgetHeader from './BudgetHeader';
import { render } from '@testing-library/react';

describe('<BudgetHeader/>', () => {
  it('matches snapshot', () => {
    const { firstChild } = render(<div><BudgetHeader groupTitle="Test"/></div>)
    expect(firstChild).toMatchSnapshot();
  })
})