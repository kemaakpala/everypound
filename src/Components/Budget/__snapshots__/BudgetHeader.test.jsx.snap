// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BudgetHeader/> matches snapshot 1`] = `undefined`;
